quic-connection-dos/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "solana-connection-dos"
+publish = false
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+clap = { workspace = true }
```

Comment:
Since this is a new tool, I would suggest to use clap-v3, better with derive features